conformance_tests/sysman/test_sysman_init/src/test_init_sysman_from_core_handle.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "gtest/gtest.h"
+#include "logging/logging.hpp"
+#include "utils/utils.hpp"
+#include "test_harness/test_harness.hpp"
+#include <level_zero/zes_api.h>
+
+namespace {
+
+TEST(SysmanInitTests,
+     GivenCoreAndSysmanInitializedThenCoreHandleReturnsUninitializedErrorForEnumFrequencyDomains) {
```

Comment:
GivenCoreAndSysmanInitializedWhenRetrievingFrequencyDomainsWithCoreHandleThenUninitializedErrorIsReturned ?

Reply:
Changing this to ```GivenSysmanInitializedFromZeInitAndZesInitThenSysmanHandleFromZeInitReturnsUninitializedErrorForEnumFrequencyDomains ```
Let me know if this is fine.

Reply:
Generally, we use core and sysman in test names instead of ZeInit & ZesInit. @AshwinKumarKulkarni can conclude on a better test name

Reply:
GivenSysmanInitialzedFromZesInitWhenCoreIsInitializedWithSysmanFlagAndzesEnumFrequencyDomainsIsCalledWithCoreHandleIThenUninitializedErrorIsReturned?

Reply:
Yeah looks good. 

minor : GivenSysmanInitialzedFromZesInit**And**CoreIsInitializedWithSysmanFlag**When**zesEnumFrequencyDomainsIsCalledWithCoreHandleThenUninitializedErrorIsReturned

just swapped "And" and "When"